edk2toollib/uefi/edk2/path_utilities.py

```diff
@@ -100,17 +100,31 @@ def __init__(self, ws: os.PathLike, package_path_list: Iterable[os.PathLike],
             package_path_packages[package_path] = \
                 [Path(p).parent for p in package_path.glob('**/*.dec')]
 
+        ignore_nested_packages = False
+        if "STUART_IGNORE_EDK_NESTED_PACKAGES" in os.environ and \
+            os.environ["STUART_IGNORE_EDK_NESTED_PACKAGES"].strip().lower() == \
```

Comment:
should the variable name mention something about temporary.  Given the pytool library can be used with more than stuart i think it makes sense to call this pytool instead of stuart. How about something like `PYTOOL_TEMPORARLY_IGNORE_NESTED_EDK_PACKAGES`

Reply:
Updated variable name in PR commit https://github.com/tianocore/edk2-pytool-library/pull/199/commits/faf600ea5c0691f6f30c5ffdc9650750c245a90c